runtime/src/main/scala/akka/grpc/internal/GrpcResponseHelpers.scala

```diff
@@ -51,11 +51,7 @@ object GrpcResponseHelpers {
     try writer.encodeDataToResponse(m.serialize(e), responseHeaders, TrailerOkAttribute)
     catch {
       case NonFatal(ex) =>
-        val trailers = GrpcEntityHelpers.handleException(ex, eHandler)
-        writer.encodeDataToResponse(
-          ByteString.empty,
-          responseHeaders,
-          Trailer(GrpcEntityHelpers.trailer(trailers.status, trailers.metadata).trailers))
```

Comment:
This is actually a bug, the empty byte string gets encoded as a gRPC message frame, so this is equivalent to sending an empty gRPC message before sending a failure message, which many/most unary clients ignore. That said, errors encountered during protobuf serialization would not be common, if they're even possible at all, so I'm not sure that this exception handler was ever triggered.